multimedia/fadecandyserver/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fadecandyserver
+PKG_VERSION:=2.1.05
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/bluewavenet/fadecandyserver/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_HASH:= #shasum -a 256 of tar.gz of source files goes here
```

Comment:
No need for this. This is known.

Reply:
It's good to improve documentation instead of comments. Remove it from Makefile, please.